frontend/src/concepts/__tests__/StartNotebookModal.spec.tsx

```diff
@@ -154,10 +159,11 @@ describe('Start Notebook modal', () => {
     const stepper = screen.getByTestId('notebook-startup-steps');
     expect(stepper).toBeTruthy();
     const steps = screen.getAllByRole('listitem');
-    expect(steps).toHaveLength(14);
-    expect(screen.getAllByTestId('step-status-Success')).toHaveLength(14);
+    expect(steps).toHaveLength(13);
+    expect(screen.getAllByTestId('step-status-Success')).toHaveLength(13);
   });
 
+  //TODO: RHOAIENG-22056 increment both .toHaveLength after reintroducting the 'pod created' event
```

Comment:
Move these comments next to the line you modified... easier to trace and follow through